src/lexedata/enrich/keep_morpheme_boundaries.py

```diff
@@ -6,12 +6,25 @@
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(
-        description="Transfer metadata of existing cognate sets to new cognatesets that may"
-        " have different IDs, but the same composition in terms of contained forms."
+        description="Transfer metadata of existing cognate sets to new cognatesets that may "
+        "have different IDs, but the same composition in terms of contained forms."
+    )
+    parser.add_argument(
+        "judgements",
+        type=Path,
+        help="Path to the csv file containing the cognate judgements.",
+    )
+    parser.add_argument(
+        "alignments",
+        type=Path,
+        help="Path to the file containing the alignments.",
+    )
+    parser.add_argument(
+        "--output-file",
+        "-o",
+        type=Path,
+        help="Path to the output file.",
```

Comment:
I see your point and this kept me thinking quite a bit. It sure would work with that script, but not with all scripts outputting something of that category. For example `cognate_code_data.py`, there are some lines of code that don't work with `sys.stdout`, so again we would end up with a different interface for some scripts.
` read_back = csv.DictReader(open(args.output_file + ".tsv"), delimiter="\t")`

Maybe it's better to stick to generating files across all scripts?

Reply:
Generally, I would think that output of a type “this is a one-time report, and it may even be empty” should be written to standardout. Output that actually generates useful data should go to files.

The cognate coder, with its clunky LingPy interface, is a special case. What it generates isn't really outputs, but temporary files to talk to LingPy, so I think it should default to a tempfile, but take an override there.